docs/dns.md

```diff
@@ -1,12 +1,10 @@
 # DNS Support
-If you point a domain or subdomain to Google Cloud DNS, you can use `kctf-setup-dns` and `kctf-chal-dns` to map *[challenge_name].yourdomain.com* to your challenge.
+If you point a domain or subdomain to Google Cloud DNS as an NS record, kCTF will create a DNS zone that maps the challenge to your challenge's IP address.
```

Comment:
cosmetic nit, you can simplify by removing the future tense without changing the meaning: "...kCTF creates a DNS zone..."